x-pack/plugins/rule_registry/server/utils/create_lifecycle_executor.ts

```diff
@@ -37,6 +37,7 @@ import {
   TAGS,
   TIMESTAMP,
   VERSION,
+  // ALERT_FLAPPING,
```

Comment:
i didn't delete this on purpose. May help the next developer to figure out how to get the path for AAD.